### PR TITLE
Web - Minor Refactor

### PR DESCRIPTION
**Pull request by Rikimbili**

### CODE REVIEW

File: web/src/hooks/useLikePost.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default function useLikePost({
 
   const { mutate: mutateLike, isLoading: isLiking } = useMutation({
     mutationKey: ["like"],
-    mutationFn: () => likePost(postId),
+    mutationFn: () => likePost(postId ?? ""),
```
Comment (rob-3) [inline web/src/hooks/useLikePost.ts:22]: No, so this just moves the same issue down a level. These functions shouldn't be interpreting `""` as a no-op imo. I would favor
```ts
mutationFn: () => {
  if (postId) likePost(postId)
}
```

**Review by rob-3 — CHANGES_REQUESTED** [review]

File: web/src/hooks/useLikePost.ts (hunk 1 above)
Comment (Rikimbili) [reply web/src/hooks/useLikePost.ts:22]: I can move it down another level

Comment (rob-3) [reply web/src/hooks/useLikePost.ts:22]: My point is just that at the first level where we know there is no `postId` and it's convenient, we should stop calling things.

Comment (rob-3) [reply web/src/hooks/useLikePost.ts:22]: Tbh, I think the `if` should go in `mutationFn`

**Comment by Rikimbili** [comment]
@rob-3 I edited the email whitelist so if you can take a look quick before approving that'd be great

**Review by rob-3 — APPROVED** [review]

**Pull request merged**